framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -391,5 +391,45 @@ bool cfg_get_unfriendly_device_max_timelife_delay_days(
     return true;
 }
 
+bool bpl_cfg_get_wpa_supplicant_ctrl_path(const std::string &iface, std::string &wpa_ctrl_path)
+{
+
+    std::string local_wpa_ctrl_path;
+    std::string param = "wpa_supplicant_ctrl_path_";
+
+    param += iface;
+    param += '=';
+
+    if (cfg_get_param(param, local_wpa_ctrl_path) < 0) {
+        MAPF_ERR("bpl_cfg_get_wpa_supplicant_ctrl_path: Failed to read: " << param);
```

Comment:
Maybe use special predefined variable `__func__` instead of manual function name duplication?
Something like:
```
__func__ + ": Failed to read: " << param
```

Reply:
Fixed